src/paho/mqtt/client.py

```diff
@@ -706,8 +706,19 @@ def connect_async(self, host, port=1883, keepalive=60, bind_address=""):
         broker. If no other messages are being exchanged, this controls the
         rate at which the client will send ping messages to the broker.
         """
-        if host is None or len(host) == 0:
-            raise ValueError('Invalid host.')
+        import re
```

Comment:
Would it be better to add this in the global imports? A user will always call this method at least once.